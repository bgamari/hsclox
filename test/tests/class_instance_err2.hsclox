class Foo {}
var f = Foo();
var a = 1;
a + b.c = 3
