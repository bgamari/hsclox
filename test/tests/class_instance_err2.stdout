[line 4] Error at '=': Invalid assignment target.
